docs/onboard-to-seed.md

```diff
@@ -0,0 +1,50 @@
+# Onboard to SEED
+
+Currently Public Officers can onboard their device to SEED by logging into the TechPass portal using non-SE GSIB machine.
+
+<div class="warn">
+ <ul>
+ <li>SEED is applicable only for <b>non-GSIB device</b>.</li>
+ <li>If you are a vendor or contractor who needs to be onboarded to SEED, contact your project manager or reporting officer.</li>
```

Comment:
as only devices are onboarded to SEED, suggesting to change as the following:

If you are a vendor or contractor, contact your project manager or reporting officer to get your SEED licence before proceeding to onboard your device to SEED.